ibis/backends/duckdb/datatypes.py

```diff
@@ -25,7 +25,7 @@
 def parse(text: str, default_decimal_parameters=(18, 3)) -> dt.DataType:
     """Parse a DuckDB type into an ibis data type."""
     primitive = (
-        spaceless_string("interval").result(dt.Interval())
+        spaceless_string("interval").result(dt.Interval('s'))
```

Comment:
I think the default interval precision for duckdb is `us`.

Reply:
Updated both the impl and test.